Documentation/UpgradeLTS/UpgradeProcess/Index.rst

```diff
@@ -63,4 +63,8 @@ If you were working with "DAM" (Digital Asset Management) in TYPO3
 Abstraction Layer).
 
 DAM and its related extensions no longer work with TYPO3 CMS 6.x.
-For this, read on in the next chapter.
\ No newline at end of file
+For this, read on in the next chapter.
+
+In TYPO3 CMS 4.5 languages could be defined by setting config.language = xy in TypoScript and having according labels in <languageKey index="xy" type="array"> in language files.
+In TYPO3 CMS 6.2 by default this is possible only for those language keys that have been defined in typo3/sysext/core/Classes/Localization/Locales.php.
```

Comment:
Links to file should ideally be given like that:

    ... that have been defined in :file:`typo3/sysext/core/Classes/Localization/Locales.php`.